bin/cqlsh.py

```diff
@@ -35,6 +35,9 @@
 from glob import glob
 from uuid import UUID
 
+import configparser
+from io import StringIO
```

Comment:
These are both stdlib, so why a separate section? Why not delete the blank line at line 37?

Reply:
plus I would move configparser right after codecs to have it ordered alhapbetically.